File: JAVA/Program03.java
////////////////////////////////////////////////////////////////////
//                                                                //
//  File name :     Program03.java                                //
//  Description :   Used to check if given number is Even or not  //
//  Author :        Pallavi Omprakash Malewar                     //
//  Date :          24/08/2025                                    //
//                                                                //
////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Number
{
    public boolean CheckEvenOdd(int iNo)
    {
        if((iNo % 2) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class Program03
{
    public static void main(String A[])
    {
        boolean bRet = false;
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number you want to check Even/Odd : ");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        bRet = nobj.CheckEvenOdd(iValue);

        if(bRet == true)
        {
            System.out.println(iValue + " is Even.");
        }
        else
        {
            System.out.println(iValue + " is Odd.");
        }

    }
}

File: CPP/Program03.cpp
///////////////////////////////////////////////////////////
//                                                       //
//  File name :     Program03.cpp                        //
//  Description :   Used to do Addition of two numbers   //
//  Author :        Pallavi Omprakash Malewar            //
//  Date :          05/09/2025                           //
//                                                       //
///////////////////////////////////////////////////////////

// Generic Program

#include<iostream>
using namespace std;

template<class T>

T Addition(T No1, T No2)
{
    T Ans;
    Ans = No1 + No2;
    return Ans;
}

int main()
{
    cout<<Addition(10, 11)<<"\n";
    cout<<Addition(10.90f, 11.70f)<<"\n";
    cout<<Addition(10.23, 11.23)<<"\n";

    return 0;
}